packages/v4/src/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8">
```

Comment:
Can we add the meta viewport tag?

Reply:
And a favicon?

Reply:
These are coming in a future PR that also adds prerendering!